JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/LinkerPreserve.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Akavache.Sqlite3;
+
+namespace JhipsterXamarin
+{
+    public static class LinkerPreserve
```

Comment:
Can you comment this class ? add link to the docs

Reply:
Commented in bb931f5